utilities/os_utils.py

```
@@ -255,7 +255,10 @@ def generate_os_matrix_dict(
 
 
 def generate_linux_instance_type_os_matrix(
-    os_name: str, preferences: list[str], arch_suffix: str | None = None
+    os_name: str,
+    preferences: list[str],
+    arch_suffix: str | None = None,
+    arch_in_preference: bool = True,
```

Ruff (0.15.12)

[warning] Boolean-typed positional argument in function definition

(FBT001)